packages/core/bootstrap/src/lib/util.ts

```diff
@@ -175,6 +175,14 @@ export const getHashOpts = (): Required<Parameters<typeof objectHash>>['1'] => (
       .includes(props),
 })
 
+export const isDebug = (): boolean => {
```

Comment:
I think we should split those two concepts, `DEBUG=true` vs `LOG_LEVEL=debug`.

We could set a production service to `LOG_LEVEL=debug` to get some more info, but we probably don't want to enable all debug options in that case. It could be beneficial to have those two as two separate concepts.